analyze/config.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+
+import sys
+from pathlib import Path
+from timestamp import Subsystem
+
+if sys.version_info[0] < 3:
+    raise "Python 3 should be used to run this script"
+
+if __name__ == '__main__':
+    sys.exit("This source file should be used only as a module")
+
+subsystem_paths = {Subsystem.LIBTEEC:
```

Comment:
These hard dependencies must be mentioned "somewhere", maybe a readme or when running `-h`.

Reply:
@jbech-linaro Sure, will add. I think I'll also add some additional app params (for each separate binary, or for the whole repo dir whatever) to make possible to overwrite it. Something like:

`./analyze --optee-working-dir=/home/user/dev/optee.repo/